README.md

```diff
@@ -79,6 +59,22 @@ make manifests
 
 More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
 
+### Test strategy
+
+Run unit tests
+```sh
+make verify
+```
+Start k3d instance locally and run integration  tests
+```sh
+make k3d-integration-test
+```
+
+Run integration tests on your current kubeconfig context
+```sh
+make verify-on-cluster
+```
```

Comment:
```suggestion
Start k3d instance locally and run integration  tests
```sh
make k3d-integration-test
```
or if you have k3d with warden
Run integration tests on your current kubeconfig context
```sh
make run-integration-tests
```
```

NOTE: something is broken with suggestion!